beacon_node/src/cli.rs

```diff
@@ -194,6 +193,14 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .help("Lighthouse by default does not discover private IP addresses. Set this flag to enable connection attempts to local addresses.")
                 .takes_value(false),
         )
+        .arg(
+            Arg::with_name("proposer-only")
+                .long("proposer-only")
+                .help("Sets this beacon node at be a block proposer only node. \
+                       This will run the beacon node in a minimal configuration that is sufficient for block production only. This flag should be used \
```

Comment:
```suggestion
                       This will run the beacon node in a minimal configuration that is sufficient for block publishing only. This flag should be used \
```

This is a minor lint that I'm mostly using to try and get Github to refresh itself. There's some weird discrepancy between this PR and https://github.com/AgeManning/lighthouse/tree/bn-proposer 🤔

Reply:
Eh, I thought I'd be able to apply this suggestion myself but Github won't let me.